xbmc/filesystem/AudioBookFileDirectory.cpp

```diff
@@ -0,0 +1,173 @@
+/*
+ *      Copyright (C) 2014 Arne Morten Kvarving
```

Comment:
Yes it is. Who ever codes it can put his name there. Although now days everything is in git repos, the notice is less important as author is known from the commit.